Source/JavaScriptCore/runtime/IntlDurationFormat.cpp

```diff
@@ -322,6 +322,26 @@ struct Element {
     std::unique_ptr<UFormattedNumber, ICUDeleter<unumf_closeResult>> m_formattedNumber;
 };
 
+enum class DurationSignType : uint8_t {
+    Negative,
+    Positive,
+    Zero,
+};
+
+// https://tc39.es/proposal-intl-duration-format/#sec-durationsign
+static DurationSignType getDurationSign(ISO8601::Duration duration)
+{
+    for (unsigned index = 0; index < numberOfTemporalUnits; ++index) {
+        TemporalUnit unit = static_cast<TemporalUnit>(index);
+        double value = duration[unit];
```

Comment:
You don't actually need to think about specific `TemporalUnit`s here. 😄 
```suggestion
    for (auto value : duration) {
```